scripts/generate_release_notes.py

```
@@ -0,0 +1,266 @@
+    return commits
+
+
+def github_request(url: str, token: str) -> tuple[list[dict], str | None]:
+    request = urllib.request.Request(
+        url,
+        headers={
+            "Accept": "application/vnd.github+json",
+            "Authorization": f"Bearer {token}",
+            "User-Agent": "crosspoint-release-notes-generator",
+        },
+    )
```

Ruff (0.15.11)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)